Widgets/Widgets/OnThisDayView.swift

```diff
@@ -29,21 +29,26 @@ struct OnThisDayColors {
 
 struct OnThisDayView: View {
     @Environment(\.widgetFamily) private var widgetSize
+    @Environment(\.sizeCategory) var textSize
     var entry: OnThisDayProvider.Entry
 
+    var doesNeedVerticalCompression: Bool {
```

Comment:
What do you think about just `needsVerticalCompression` for these?

Reply:
I'm in the habit of having booleans get an `is`/`was`/'does`/etc. prefix, but `needs` provides the same clarity I think. Good call. Updating now.